### DISCUSSION TITLE
Auto-width table columns

### DISCUSSION THREAD

**Discussion by gabrielgasp** [beekeeper-studio/beekeeper-studio#1594] [Q&A] [answered]
Is there an option or a button that allows me to resize all columns' width in a table to best fit with the content?

The only way I can find is to double-click when the icon to resize a column appears while hovering, but doing this for each column in the table is a pain for tables with lots of columns.

**Comment by OrionFOTL** [comment, marked as answer]
Yes, right-click the column header.

**Reply by gabrielgasp** [reply]
Thanks!
